setup/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+services:
+  ovehub-ove-asset-db:
+    image: mariadb
+    ports:
+    - "3306:3306"
+    volumes:
+      - "ovehub-ove-asset-db-data:/var/lib/mysql"
+    environment:
+      MYSQL_RANDOM_ROOT_PASSWORD: "yes"
+      MYSQL_DATABASE: ""
+      MYSQL_USER: ""
+      MYSQL_PASSWORD: ""
+
+  ovehub-ove-asset-manager:
+    image: ovehub/ove-asset-manager:latest
+    depends_on:
+    - ovehub-ove-asset-db
+    ports:
+    - "80:80"
```

Comment:
Does the asset manager run on port 80? This does not seem correct

Reply:
By default the service runs on port 80. For docker we leave that set to the default and we use docker to remap the port to a different value. In this case the port is remapped to 80 again for  convenience.

Reply:
@oserban the port mapping (defaults I mean) is a bit concerning. I fixed the same previously, https://github.com/ove/ove-asset-services/pull/23/files and it seems we are back to square one with this. Unless we stick the convention people won't be able to start these out of the box. Can we fix?